src/main/java/com/glovo/ttglovo/securityManagement/appuser/AppUser.java

```diff
@@ -0,0 +1,172 @@
+package com.glovo.ttglovo.securityManagement.appuser;
+
+import com.glovo.ttglovo.cart.CartItem;
+import lombok.*;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import java.util.*;
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "AppUser")
```

Comment:
If You're not using a different name than class name, no need for this annotation

Reply:
We named "AppUser" because "User" is a build in Class and sometimes Hibernate doesn't know what to map and we get a big error